src/passa/cli/add.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding=utf-8 -*-
+
```

Comment:
I know I disagreed with you about putting this semantic in pipenv— it makes a lot more sense with a single purpose library that doesn’t actually install things!

Reply:
We’ll need to discuss this. I thought a bit about this, and it isn’t really feasible to implement this in Pipenv (at least with my current approach). Passa needs to be run by Pipenv with a child process (like the current resolver.py), so no matter where we put the implementation, it would just be a few files calling the Python interface. Putting them here at least making them easier to test…

Reply:
Yeah I agree with this implementation is what I'm saying -- since this is separated and not a kind oif ad-hoc piece of pipenv, it makes a lot more sense to me to use the `add` and `remove` semantics in here vs `uninstall` and `install` which is what we are using in pipenv

The separation here is good -- we should keep it like this -- that's what i'm saying